setup/resources/user-workloads.yaml

```diff
@@ -0,0 +1,157 @@
+kind: Template
```

Comment:
The README contains a reference to the current sample resource: https://github.com/codeready-toolchain/toolchain-e2e/tree/master/setup/sample so you should probably replace it or change the README link.
You could also add information that the file should contain the OS template

Reply:
done in 00c8f2e and 86b510c